src/sentry/conf/server.py

```diff
@@ -373,7 +373,7 @@ def env(key, default="", type=None):
 STATIC_URL = "/_static/{version}/"
 # webpack assets live at a different URL that is unversioned
 # as we configure webpack to include file content based hash in the filename
-STATIC_UNVERSIONED_URL = "/_static/dist/"
+STATIC_WEBPACK_URL = "/_static/dist/"
```

Comment:
how about `client_app` instead of `webpack`?

Reply:
@EvanPurkhiser I like that direction more since it's not tied to a specific impl/vendor. what about using `spa`? or is that too obscure?

Reply:
What about even just “frontend”

Reply:
I like frontend, would people get it confused with non-webpack'd assets that get served?

Reply:
I think it might be safe as long as we keep the comment above it

`STATIC_FRONTEND_APP_ASSETS` even, though it's a tad long

Reply:
Yeah although this will be used throughout the app as well. I would prefer a single word, but `frontend app` is more clear

Reply:
I don't _hate_ `spa` but it's more cryptic

Reply:
`STATIC_SINGLE_PAGE_APPLICATION_APP_ASSET_URL`

Reply:
:shipit: